readme.md

```diff
@@ -17,221 +16,31 @@ MISO: An open source LIMS for small-to-large scale sequencing centres
-
-> NB: You shouldn't need to change the miso.properties file as long as the /storage/miso directory exists on your filesystem and is writeable!
-
-4.3 ) Setting up userspace properties
-
-MISO achieves userspace configuration via extra properties files kept in the storage directory specified by the `miso.baseDirectory`
-property (see 2.2 above). Default versions of these files are available here:
-
-https://repos.tgac.ac.uk/miso/latest/miso_userspace_properties.tar.gz
-
-Unpack this file to your MISO storage directory, which again is `/storage/miso` by default. You should see 4 files:
-
-* issuetracker.properties - settings for an issue tracking system, such as JIRA or RT.
-* mail.properties - email settings so that MISO can send emails to users.
-* security.properties - properties to set the security environment (see 2.4 below).
-* submission.properties - properties to set the submission environment (see 2.5 below).
-
-4.4 ) Setting the security environment
-
-MISO can use either LDAP or SQL as an authentication mechanism, and will need to know where your chosen method resides. These
-properties need to be set before the container is started, and relate to properties files stored on disk.
-
-Firstly, the security options for both auth systems are specified in the `security.properties` file. If you are using JDBC,
-then it is unlikely that you will need to change any options, but the LDAP setup is far more complex.
-Please ask your sysadmin to match up the MISO LDAP options with your local LDAP server options.
-
-Secondly, the `security.method` environment property relates to the security mechanism, i.e. 'ldap' or 'jdbc', where 'ldap' auths
-against an LDAP server, and 'jdbc' auths against a local database (usually the MISO database itself). Please use 'jdbc' if you
-are unsure. There is a default admin user that is shipped with the MISO database, and the username/password login is admin/admin.
-
-To set the `security.method` property, use local environment variables on Tomcat startup. Please see the "Starting MISO" section below.
-
-> NB: You are likely to see startup errors like the one below if you do not set this property, and start Tomcat as outlined in section 3!
-
-> 20-Oct-2010 11:08:28 org.apache.catalina.core.StandardContext start
-
-> SEVERE: Error listenerStart
-
-> 20-Oct-2010 11:08:28 org.apache.catalina.core.StandardContext start
-
-> SEVERE: Context [/tgac] startup failed due to previous errors
-
-4.5 ) Submission properties
 
-MISO is able to submit sequence data to the major sequence archives, i.e. the SRA at the EBI, and the EMBL GenBank. Currently,
-only ERA submissions are supported out-of-the-box. The settings for these services are specified in the `submission.properties` file.
+View [Full MISO Documentation](https://documentation.tgac.ac.uk/display/MISO/MISO+Documentation+Home)
 
-5 ) STARTING MISO
------------------
+## Running an Instance of MISO
```

Comment:
Here's the text from the github version:

```
The simplest way to get MISO up and running quickly is to use our virtual machine image with everything preinstalled. A VirtualBox instance is available with a self-contained MISO installation and MySQL database server, fully configured and can be used out-of-the-box.

To use it:

1.  Download VirtualBox software from: [https://www.virtualbox.org/](https://www.virtualbox.org/)
2.  Download the MISO OVA: [https://repos.tgac.ac.uk/miso/latest/MISO_Image.ova](https://repos.tgac.ac.uk/miso/latest/MISO_Image.ova)
3.  Import this OVA file into VirtualBox via the _File, Import Appliance_ option.
    1. You may have to select **Bridged Adaptor** instead of **NAT** in the network settings
3.  Start the instance. Starting the appliance will result in an Ubuntu virtual machine complete with all MISO’s prerequisites pre-installed. The login for the virtual image is **miso / misoadmin**.
4.  Start MISO server by using ./misoStart.sh and then open the browser and go http://localhost:8090/
5.  Log in to MISO using admin / admin. Only some simple configuration for certain elements is required, such as Sequencer References, Printers and any additional users and groups.

For more information about the VirtualBox image, see the [MISO 0.2.0 User Manual]({{ site.baseurl }}{% post_url 2016-01-12-user-manual %}).
```